python/py-keep/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-keep
+version             2.10.1
+license             MIT
+supported_archs     noarch
+maintainers         nomaintainer
+
+description         A Meta CLI toolkit
+long_description    A Meta CLI toolkit: your personal shell command keeper
+homepage            https://pypi.org/project/keep/
```

Comment:
please use the "real" upstream website: https://github.com/orkohunter/keep

Reply:
@reneeotten Just for homepage link, not for the source?

Reply:
Yes, please: for the source we prefer PyPI in case of Python ports; but for the website one typically takes GitHub - the latter  is perhaps a personal opinion ;)